portal-web/test/functional/com/liferay/portalweb/macros/ABTest.macro

```diff
@@ -248,7 +250,7 @@ definition {
 			key_text = "Publish",
 			locator1 = "Button#ANY");
 
-		Alert.viewSuccessMessageText(successMessage = "Success:${variantName} was published successfully.");
+		AssertVisible(locator1 = "ABTest#IMAGE_EMPTY_STATE");
```

Comment:
Hey @marcosapmf, I think it is not correct. After publishing a variant, then the success alert should appear, why do you change to check if the empty state will appear?

Reply:
@ViniLopes20, the success message does appear. However, the page is reloaded before Poshi could check the alert. I talked to @interaminense about this, and the best option was finding another way to assert the variant was published. One way is guaranteeing that we have the empty state in AB test side panel :)

Reply:
Hey @marcosapmf, the purpose of this `Alert.viewSuccessMessageText` is not to guarantee that the variant was published (we do this by checking the version that is appearing on the page). This alert is to ensure that the alert is displaying the feedback of the page to the user, if this is not showing then this is a bug.

This change should not be made

Reply:
Vini, this is a usability issue, currently, after the user publishes, the page has to be redirected to the published variant, so the alert no longer makes sense in this case, I would even talk to Deborah about removing this alert.

Reply:
Hey @ViniLopes20, in light of what you and Adriano said, we can just remove this alert assertion for now, right?

Reply:
Hey @marcosapmf, Let's see this on your machine, I'm validating it here on mine and I don't see the page being loaded again and removing the success alert.